src/Pickles/Pickles.DocumentationBuilders.Json/Mapper/TableRowToJsonTableHeaderMapper.cs

```diff
@@ -0,0 +1,46 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="TableRowToJsonTableRowMapper.cs" company="PicklesDoc">
+//  Copyright 2011 Jeffrey Cameron
+//  Copyright 2012-present PicklesDoc team and community contributors
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+using System;
+
+using PicklesDoc.Pickles.ObjectModel;
+
+namespace PicklesDoc.Pickles.DocumentationBuilders.Json.Mapper
+{
+    public class TableRowToJsonTableHeaderMapper
+    {
+        private readonly TestResultToJsonTestResultMapper testResultMapper;
```

Comment:
do we need the testResultMapper? It does not seem to be used.

Reply:
@dirkrombauts : it is definitely used in the code:
![image](https://user-images.githubusercontent.com/14995941/35408570-8c14a644-0274-11e8-8eac-491cc2a0f581.png)
![image](https://user-images.githubusercontent.com/14995941/35408534-5be466ee-0274-11e8-9790-e4e0a7bdb524.png)
![image](https://user-images.githubusercontent.com/14995941/35408556-6d968a34-0274-11e8-9a2e-13f3501b1634.png)